sqldelight-gradle-plugin/src/main/kotlin/app/cash/sqldelight/gradle/android/PackageName.kt

```diff
@@ -29,6 +29,9 @@ internal fun Project.packageName(): String {
 internal fun Project.sqliteVersion(): String? {
   val androidExtension = extensions.getByType(BaseExtension::class.java)
   val minSdk = androidExtension.defaultConfig.minSdk ?: return null
+
+  // Mapping available at https://developer.android.com/reference/android/database/sqlite/package-summary.
+  if (minSdk >= 34) return "app.cash.sqldelight:sqlite-3-38-dialect:$VERSION"
```

Comment:
Filed https://issuetracker.google.com/issues/286399699 to get the table updated for API 34.